src/main/java/fact/io/hdureader/ZFITSHeapReader.java

```diff
@@ -130,17 +132,58 @@ public static ZFITSHeapReader forTable(BinTable binTable){
      *
      * The old ZFits implementation by M.Bulinski ignored the problem by default.
      *
-     * @param ignoreWrongTileHeader if true, ognores wrong tileheader information.
+     * @param ignoreWrongTileHeader if true, ignores wrong tileheader information.
      * @return a map containing the the data from the rows columns.
      * @throws NoSuchElementException iff hasNext() is false
      */
     public OptionalTypesMap<String, Serializable> getNextRow(boolean ignoreWrongTileHeader) throws IOException {
```

Comment:
So the problem is that there can be now many rows in one tile right? Why do we need to read many tiles then? Or is the problem that one row is split over many tiles?